requirements-dev.in

```diff
@@ -3,7 +3,7 @@ black>=23.7.0
 coverage>=7.2.2
 isort>=5.12.0
 mypy>=1.2.0
-nose>=1.3.7
+nose-py3>=1.6.3 # bug in regular nose here: https://github.com/nose-devs/nose/issues/1099#issuecomment-577647469
```

Comment:
I think we shouldn't use `nose` anymore [anyways](https://github.com/broadinstitute/seqr-loading-pipelines/issues/600)